index.bs

```diff
@@ -5787,6 +5787,18 @@ The attestation certificate MUST have the following fields/extensions:
     are both OPTIONAL as the status of many attestation certificates is available through authenticator metadata services.
     See, for example, the FIDO Metadata Service [[FIDOMetadataService]].
 
+### Certificate Requirements for Enterprise Packed Attestation Statements ### {#sctn-enterprise-packed-attestation-cert-requirements}
+
+There are two potential sources for Enterprise Attestestations in Packed format:
+
+1. From a hardware device which is manufactured with a device-specific attestation statement and key
+2. Via a provisioned attestation statement, such as with a platform authenticator configured via managed policy
+
+For attestation statements provisioned at manufacturing, the Extension OID `1.3.6.1.4.1.45724.1.1.2` (`id-fido-gen-ce-sernum`) 
+MUST be present, containing a unique serial number for the device as an OCTET STRING. The extension MUST NOT be marked as critical.
```

Comment:
Does it matter how the serial number (if it is in fact an integer) is encoded as an OCTET STRING? If so, we should define that encoding (probably two's complement, but what byte order?). If not, maybe we should describe it as "containing a unique device identifier" instead of as a number, to make it clear that it's an opaque byte array whose internal structure is undefined?

Also, an example like the one for `id-fido-gen-ce-aaguid` would be helpful to point out that the value needs to be two nested OCTET STRING values.

Reply:
The double octet string is just because there is inner ASN in the field that way you could embed a struct or enum into the oid field. It's similar in LDAP in a few places. The outer octet string is parsed as a new blob of asn and the inner can be any ASN value as a result. That way the oid tells a client how to parse/what to expect of the inner ASN blob.